src/main/java/executor/service/service/impl/ExecutionServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package executor.service.service.impl;
+
+import executor.service.model.ProxyConfigHolder;
+import executor.service.model.Scenario;
+import executor.service.model.WebDriverConfig;
+import executor.service.service.ExecutionService;
+import executor.service.service.ScenarioExecutor;
+import executor.service.service.WebDriverInitializer;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExecutionServiceImpl implements ExecutionService {
```

Comment:
This class is like mine, see ExecutionService, pull-request [#46](https://github.com/GFLCourses5/worker/pull/46). 
Please, approve ExecutionService, pull-request [#46](https://github.com/GFLCourses5/worker/pull/46).